File: api/v2010/message/create-example-2/create-example-2.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

message = @client.messages.create(
                             body: 'Hello there!',
                             from: '+15555555555',
                             media_url: 'https://demo.twilio.com/owl.png',
                             to: '+12316851234'
                           )

puts message.sid

File: api/v2010/message/create-example-2/create-example-2.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Linq;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var mediaUrl = new [] {
            new Uri("https://demo.twilio.com/owl.png")
        }.ToList();

        var message = MessageResource.Create(
            body: "Hello there!",
            from: new Twilio.Types.PhoneNumber("+15555555555"),
            mediaUrl: mediaUrl,
            to: new Twilio.Types.PhoneNumber("+12316851234")
        );

        Console.WriteLine(message.Sid);
    }
}
